internal/run.bzl

```diff
@@ -250,9 +262,11 @@ _postcss_multi_run = rule(
         "sourcemap": attr.bool(default = False),
         "data": attr.label_list(allow_files = True),
         "named_data": attr.label_keyed_string_dict(allow_files = True),
+        "plugins_info": attr.label(
+            cfg = "exec",
+        ),
```

Comment:
Can you require this to have a `PostcssPluginInfo` provider? If not, can you throw an error if it's passed but doesn't have one in practice?

Reply:
Didn't realise this was a thing, thanks!